spec/lib/dispatch-rider/dispatcher_spec.rb

```diff
@@ -55,7 +55,7 @@ def process(params)
     end
 
     context "when the handler returns false" do
-      let(:message){ DispatchRider::Message.new(:subject => "handler_that_returns_false", :body => { :do_throw_something => true }) }
+      let(:message) { DispatchRider::Message.new(:subject => "handler_that_returns_false", :body => {:do_throw_something => true}) }
```

Comment:
Line is too long. [132/120]<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.